flink-runtime/src/test/java/org/apache/flink/runtime/state/IncrementalKeyedStateHandleTest.java

```diff
@@ -187,7 +188,7 @@ public void testSharedStateDeRegistration() throws Exception {
 
 	private static IncrementalKeyedStateHandle create(Random rnd) {
 		return new IncrementalKeyedStateHandle(
-			"test",
+			UUID.nameUUIDFromBytes("test".getBytes()),
```

Comment:
Let's set the right example for everyone that "learns by reading" other code.
Also, we have seen cases where tests were unstable when executed on an Italian Laptop because of different default locales and charsets ;-)